interface/src/app/volcano-plot/view.js

```diff
@@ -0,0 +1,34 @@
+angular.module('adage.volcano-plot.view', [
+  'ui.router',
+  'adage.analyze.sampleBin',
+  'adage.volcano-plot'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('volcano', {
+    url: '/volcano',
+    views: {
+      main: {
+        templateUrl: 'volcano-plot/view.tpl.html',
+        controller: 'VolcanoPlotViewCtrl as ctrl'
+      }
+    },
+    data: {pageTitle: 'Volcano Plot'}
+  });
+}])
+
+.controller('VolcanoPlotViewCtrl', ['SampleBin', '$stateParams',
+  // TODO make use of $stateParams (pulling from SampleBin for initial tests,
+  //      but the right way to do this is to refactor so view.js pulls params
+  //      for sample-group-a and sample-group-b and does what's necessary to
+  //      make a plot from those lists)
+  function VolcanoPlotViewCtrl(SampleBin, $stateParams) {
```

Comment:
It should be used for this view, but it's not in there yet because of the trouble I ran into with the volcano plot selection. I'll put a comment in here to indicate how it will be used.

- [ ] add a note about why $stateParams is injected into `volcano-plot/view.js`.